src/styles/kelsier.css

```
@@ -1,28 +1,96 @@
+	--k-surface: color-mix(in srgb, var(--k-text) 2.5%, transparent);
+	--k-track: color-mix(in srgb, var(--k-text) 7%, transparent);
+	--k-nav-bg: color-mix(in srgb, var(--k-void) 90%, transparent);
+	--k-star: var(--k-text);
+	--k-on-accent: #07090e;
 	--k-shadow: 0 18px 40px rgba(0, 0, 0, 0.26);
+	--k-data-1: #3b5fa0;
+	--k-data-2: #3a7a68;
+	--k-data-3: #8c5fa0;
+	--k-data-4: var(--k-accent);
+	--k-data-5: #c0544a;
 	--font-display-k: "Cormorant Garamond", Georgia, serif;
```

Stylelint (17.14.1)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)